### PR TITLE
feat(NX-000): Animated markers transition

### PR DESCRIPTION
**Pull request by pietroid**
Enabling animation of markers when adding and removing markers for a more smooth UX.

https://user-images.githubusercontent.com/22605271/228967492-3c3d79ef-d0b5-4f9b-9725-b527d09064e9.mp4

### CODE REVIEW

File: packages/google_maps_flutter/google_maps_flutter/pubspec.yaml [hunk 1]
```diff
@@ -23,19 +23,19 @@ dependencies:
   google_maps_flutter_android: 
     git:
       url: https://github.com/fulpismo/plugins.git
-      ref: v2.2.3-fork.4
+      ref: feat/animated-markers
```
Comment (fulpismo) [inline packages/google_maps_flutter/google_maps_flutter/pubspec.yaml]: minor issue for now, but what do you think about using the same naming conventions from the tags in this PR as well?
you could name it `v2.2.3-fork.5` and after merging it, we create a new release with the same name (and we don't have to change it in another commit or PR)
alternatively you can also change it to `main-fork` so all packages point to the target branch when using this version temporarily.

Comment (pietroid) [reply packages/google_maps_flutter/google_maps_flutter/pubspec.yaml]: Oh, I will change that! Thanks!

Comment (pietroid) [inline packages/google_maps_flutter/google_maps_flutter/pubspec.yaml]:
```suggestion
      ref: v2.2.3-fork.5
```

File: packages/google_maps_flutter/google_maps_flutter/pubspec.yaml [hunk 2]
```diff
@@ -23,19 +23,19 @@ dependencies:
...
       path: packages/google_maps_flutter/google_maps_flutter_android    
     
   google_maps_flutter_ios: 
     git:
       url: https://github.com/fulpismo/plugins.git
-      ref: v2.2.3-fork.4
+      ref: feat/animated-markers
```
Comment (pietroid) [inline packages/google_maps_flutter/google_maps_flutter/pubspec.yaml]:
```suggestion
      ref: v2.2.3-fork.5
```

File: packages/google_maps_flutter/google_maps_flutter/pubspec.yaml [hunk 3]
```diff
@@ -23,19 +23,19 @@ dependencies:
...
       path: packages/google_maps_flutter/google_maps_flutter_ios    
 
   google_maps_flutter_platform_interface: 
     git:
       url: https://github.com/fulpismo/plugins.git
-      ref: v2.2.3-fork.4
+      ref: feat/animated-markers
```
Comment (pietroid) [inline packages/google_maps_flutter/google_maps_flutter/pubspec.yaml]:
```suggestion
      ref: v2.2.3-fork.5
```

File: packages/google_maps_flutter/google_maps_flutter_android/pubspec.yaml [hunk 4]
```diff
@@ -25,7 +25,7 @@ dependencies:
   google_maps_flutter_platform_interface:
     git:
       url: https://github.com/fulpismo/plugins.git
-      ref: v2.2.3-fork.4
+      ref: feat/animated-markers
```
Comment (pietroid) [inline packages/google_maps_flutter/google_maps_flutter_android/pubspec.yaml]:
```suggestion
      ref: v2.2.3-fork.5
```

File: packages/google_maps_flutter/google_maps_flutter_ios/pubspec.yaml [hunk 5]
```diff
@@ -23,7 +23,7 @@ dependencies:
   google_maps_flutter_platform_interface:
     git:
       url: https://github.com/fulpismo/plugins.git
-      ref: v2.2.3-fork.4
+      ref: feat/animated-markers
```
Comment (pietroid) [inline packages/google_maps_flutter/google_maps_flutter_ios/pubspec.yaml]:
```suggestion
      ref: v2.2.3-fork.5
```

**Review by fulpismo — APPROVED** [review]

**Pull request merged**